core/backend/src/IModelDb.ts

```diff
@@ -1076,6 +1079,19 @@ export abstract class IModelDb extends IModel {
     return this._classMetaDataRegistry;
   }
 
+  public get schemaContext(): SchemaContext {
```

Comment:
Add documentation now so we don't have a debt of docs to fill out at the end.  It will also help other people not familiar with ecschema-metadata understand the changes

Reply:
done